spec/ginkgo_spec.vim

```diff
@@ -0,0 +1,73 @@
+source spec/support/helpers.vim
+
+describe "Ginkgo"
+
+  before
+    let g:test#go#runner = 'ginkgo'
```

Comment:
I couldn't find a way to parametrize spec in vspec.

I removed `let g:test#go#runner = 'ginkgo'` here so it tests the scenario when the user does not set `go#runner`, which is the default behavior. One more test is added to make sure TestNearest is not broken by the `search` call in `test_file`